src/test/java/teammates/test/cases/webapi/UpdateStudentProfileActionTest.java

```diff
@@ -49,9 +50,10 @@ private void testActionWithInvalidParameters(AccountAttributes student) throws E
         loginAsStudent(student.googleId);
```

Comment:
Can these tests be separated and marked `@Test` individually?
```java
testActionWithInvalidParameters(student1);
        testActionSuccess(student1, "Typical Case");
        testActionForbidden(student1, student2, "Forbidden Case");
        testActionInMasqueradeMode(student1);
```

Reply:
Currently it has only one marked `@Test` and calls many sub tests. So I just separate them into two functions?

Reply:
See if you can do that.